.gitignore

```diff
@@ -15,6 +15,9 @@ certs
 *.[568vq]
 [568vq].out
 
+# Override the *.[568vq] or else directories like x.x.5/ will be excluded.
```

Comment:
Just remove the rule above. We picked it up from github's default .gitignore for go projects, but I don't think we really need it. It matches intermediate files created by older versions of the go compiler (or at least I've never seen them in practice).